fe/fe-core/src/main/java/com/starrocks/alter/AlterJobMgr.java

```diff
@@ -181,9 +181,15 @@ public void processCreateMaterializedView(CreateMaterializedViewStmt stmt)
             if (table == null) {
                 throw new DdlException("create materialized failed. table:" + tableName + " not exist");
             }
+            if (table.isCloudNativeTable()) {
+                throw new DdlException("Creating synchronous materialized view(rollup) is not supported in " +
+                        "shared data clusters.\nPlease use asynchronous materialized view instead.\n" +
+                        "Refer to https://docs.starrocks.io/en-us/latest/sql-reference/sql-statements" +
+                        "/data-definition/CREATE%20MATERIALIZED%20VIEW#asynchronous-materialized-view for details.");
```

Comment:
The error is synchronous MV, the doc is asynchronous MV.

Reply:
The link is used to tell user how to create asynchronous materialized view.

Reply:
OK